recipes/log4cxx/all/patches/0002-find-apr.patch

```diff
@@ -0,0 +1,17 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -41,10 +41,12 @@ if(NOT BUILD_SHARED_LIBS)
+ endif()
+ 
+ # Find Apache Runtime
+-find_package(APR REQUIRED)
++find_package(apr REQUIRED)
++set(APR_LIBRARIES apr::apr)
+ 
+ # Find Apache Runtime Utilities
+-find_package(APR-Util REQUIRED)
++find_package(apr-util REQUIRED)
```

Comment:
This can be fixed in the recipe with CMakeDeps and set_property. The patch is not required

Reply:
Can you tell me how? I tried:

  tc = CMakeDeps(self)
  if Version(self.version) < "0.13.0":
      tc.set_property("expat::expat", "cmake_target_name", "EXPAT::EXPAT")
  tc.generate()

Reply:
https://docs.conan.io/2/reference/tools/cmake/cmakedeps.html?highlight=set_property#overwrite-properties-from-the-consumer-side-using-cmakedeps-set-property

https://github.com/conan-io/conan/blob/048295bd4bcc6be9692dab92e232248b67f9541c/conans/test/integration/toolchains/cmake/cmakedeps/test_cmakedeps.py#L304

```py
def generate(self):
    deps = CMakeDeps(self)
    deps.set_property("expat", "cmake_target_name", "EXPAT::EXPAT")
    deps.generate()
```

Reply:
This can be used here for apr and apr-utils not only expat. 
This patch is not required